torch/csrc/profiler/collection.cpp

```diff
@@ -1406,7 +1406,7 @@ RecordQueue::getRecords(
     for (const auto& i : python_tracer_->getEvents(
              converter,
              python_enters,
-             static_cast<c10::time_t>(end_time_us * 1000))) {
+             static_cast<c10::time_t>(end_time_ns))) {
```

Comment:
```suggestion
             converter, python_enters, static_cast<c10::time_t>(end_time_ns))) {
```
looks like lintrunner needs a fix like so.